libs/set/set.go

```diff
@@ -0,0 +1,75 @@
+package set
+
+import (
+	"fmt"
+
+	"golang.org/x/exp/maps"
```

Comment:
Can use stdlib version

Reply:
maps.Values and maps.Keys are not yet part of stdlib

https://github.com/golang/go/issues/61613